ci/env/install-core-prerelease-dependencies.sh

```diff
@@ -4,7 +4,5 @@ set -e
 
 # install all unbounded dependencies in setup.py and any additional test dependencies
 # for the min build for ray core
-# TODO(scv119) reenable grpcio once https://github.com/grpc/grpc/issues/31885 is fixed.
-# TODO(scv119) reenable jsonschema once https://github.com/ray-project/ray/issues/33411 is fixed.
-DEPS=(requests protobuf pytest-httpserver==1.1.3)
+DEPS=(requests protobuf pytest-httpserver==1.1.3 grpcio==1.74.0 jsonschema==4.23.0)
```

Comment:
The IPPR implementation needs grpcio and jsonschema. The issue of jsonschema has been closed, and the issue of grpcio has been fixed by 1.74, according to https://github.com/grpc/grpc/issues/31885#issuecomment-3020611208.